### ISSUE TITLE
Wrong Learning Rate Conditions

### ISSUE THREAD

**Issue by kyleosung** [kyleosung/StepByStep-AdaptiveGDForLipschitzNNs_DEV#9]
I think we are using the wrong learning rate conditions. Can you instead make sure that we are using this rule:

    \begin{equation*}
        \begin{alignedat}{3}
            \alpha_t^{W^{[m]}} & \leq  
                \dfrac{C_{W^{[m]}} g(t)}{\sum\limits_{n=1}^N \left\| (\nabla_{W^{[m]}} f_{\theta_{t-1}}(\x_n))^\top  \Big [f_{\theta_{t-1}}(\x_n) - y_n \Big ] \right\|_2}
            \\
            \alpha_t^{b^{[m]}} & \leq  
                \dfrac{C_{b^{[m]}} g(t)}{\sum\limits_{n=1}^N \left\| (\nabla_{b^{[m]}} f_{\theta_{t-1}}(\x_n))^\top  \Big [f_{\theta_{t-1}}(\x_n) - y_n \Big ] \right\|_2}
            .
        \end{alignedat}
    \end{equation*}

Where we set the learning rates to be the right side, except where we add a small positive constant to the denominator?

We should also make it clear through the experiments when and if we use a constant learning rate before the hybrid regime begins. We should also add a red dashed vertical line to all plots at T (usually 500) epochs to show when we start training with the GD LR decay scheme.
